mcodingbot/plugins/python.py

```diff
@@ -22,9 +23,14 @@ class PEPCommand:
     pep_number = crescent.option(
         int, "The PEP number.", name="pep-number", min_value=0, max_value=9999
     )
+    hide_embed = crescent.option(
```

Comment:
Imo, it should be the opposite (aka `show_embed` defaulting to `True`) but that's up to code style

Reply:
For the command option, I agree. I'll change that.